src/test/kotlin/no/nav/familie/ef/sak/infrastruktur/config/OppgaveClientMock.kt

```diff
@@ -57,44 +56,12 @@ class OppgaveClientMock {
             if (mappeRequest.enhetsnr == "4489") {
                 FinnMappeResponseDto(
                     antallTreffTotalt = 2,
-                    mapper = listOf(
```

Comment:
Husker ikke helt i farta, men tror noe kommer til å krasje lokalt hvis vi ikke har noen mapper?

Send til totrinns -> Oppretter oppgave og ønsker å putte den i `Godkjenne vedtak`-mappa?
Fint å ha noen mapper lokalt også. Så Klar til behandling og Godkjenne vedtak er nok greit bare for å se at ting fungerer.

Reply:
Kunne vi beholdt mappene, men bare fjernet "EF Sak -" - ikke fordi det er veldig viktig at dette er helt prodlikt, men (mangler argument her 😛)

Reply:
Litt dét jeg også ønsker. Samt unngå at noe guff smeller unødig lokalt

Reply:
Ah, ja, det er vel greit å ha et par av disse. Fjerner bare "EF Sak -" prefiks da